src/cmd/build.js

```diff
@@ -45,11 +45,20 @@ export type LocalizedNameParams = {
   manifestData: ExtensionManifest,
 }
 
+// This defines the _locales/messages.json type. See:
+// https://developer.mozilla.org/en-US/Add-ons/WebExtensions/Internationalization#Providing_localized_strings_in__locales
+type LocalizedMessageData = {
+  [messageName: string]: {
+    description: string,
+    message: string,
+  },
+}
```

Comment:
nit (and probably something that would be better handled in a follow up), we should decide if we want to close the type declarations with `;` or if we want to omit it, and keep the style consistently (and checking if any of the flow eslint plugins can help us to enforce the style that we decide to use).